MIGRATE.md

```diff
@@ -1,15 +1,29 @@
-## webpack-migrate
-
-The `migrate` feature eases the transition from [version 1](http://webpack.github.io/docs/) to [version 2](https://gist.github.com/sokra/27b24881210b56bbaff7). `migrate`
-also allows users to switch to the new version of webpack without having to extensively [refactor](https://webpack.js.org/guides/migrating/).
-
-### Usage
-To use `migrate`, run the following command, with the value of `<config>` being a path to an existing webpack configuration file
+# webpack-cli migrate
+
+The `webpack-cli migrate` feature eases the transition from version 1 to version 2. `webpack-cli migrate`
+also allows users to switch to the new version of webpack without having to extensively.
+## Installation
+> Requires installation of `webpack` and `webpack-cli`
+### A. Local Setup
+```shell
+$ npm install --save-dev @webpack-cli/migrate
+```
+### B. Global Setup
+```shell
+$ npm install -g @webpack-cli/migrate
+```
 
+## Usage
+To use `webpack-cli migrate`, run the following command, with the value of `<config>` being a path to an existing webpack configuration file
+### A. Local Setup
+```bash
+npx webpack-cli migrate <config>
```

Comment:
I'd add also a real example
```bash
npx webpacli-cli migrate webpack.config.js
```

Reply:
See if it's now fine @ematipico after fd7bd9f.